packages/connect/src/connect.ts

```diff
@@ -13,18 +13,22 @@ import ConnectorTheGraph, {
 } from '@aragon/connect-thegraph'
 import { Network } from '@aragon/connect-types'
 
-type ConnectOptions = {
+export type ConnectOptionsResolveIpfs = (
+  ipfsIdentifier: string,
+  path: string
+) => string
+
+export type ConnectOptions = {
   readProvider?: ethers.providers.Provider
```

Comment:
I want to double-check if we still want to call this `readProvider` or just `provider`. Even though we only use it to read from the blockchain.

Reply:
Yes I don’t like `readProvider` much, let’s use `provider` (or `ethereum`?) and use the documentation to communicate that it’s read only.

Reply:
Added it here https://github.com/aragon/connect/issues/109